kamelet-support/src/main/java/io/kaoto/backend/api/service/step/parser/kamelet/KameletStepParserService.java

```diff
@@ -296,6 +299,22 @@ private Step processDefinedStep(final SetHeaderFlowStep step) {
         return res;
     }
 
+    private Step processDefinedStep(final TransformFlowStep step) {
+        Step res = catalog.getReadOnlyCatalog().searchStepByName("transform");
```

Comment:
This returns null ATM

Reply:
Right now it loads whatever is on the latest main branch in https://github.com/KaotoIO/camel-component-metadata

Reply:
There's probably a hundred better ways to do this, but at this point you should add a file like https://github.com/KaotoIO/camel-component-metadata/blob/main/set-body.camel-component.yaml

Reply:
https://github.com/KaotoIO/camel-component-metadata/blob/main/transform.camel-component.yaml